packages/admin-panel-server/src/viz-builder/dashboardVisualisation/combineDashboardVisualisation.ts

```diff
@@ -24,20 +18,22 @@ const getPresentation = (dashboardItem: DashboardItem, report: Report | LegacyRe
   const { type, name, ...config } = dashboardItemConfig;
 
   const presentation: Record<string, unknown> = { type, ...config };
-  if (!dashboardItem.legacy) {
+  if (!dashboardItem.legacy && 'output' in reportConfig) {
```

Comment:
Having typescript in this package was an absolute godsend! It would have been sooo bug prone to try and make such a change without type safety 😄